test/tutorial/transaction/transaction.cpp

```diff
@@ -76,6 +77,17 @@ class TransactionCallbacks : public libdnf::rpm::TransactionCallbacks {
     }
 };
 
+// Verify the transaction (i.e. run it with RPMTRANS_FLAG_TEST)
+//
+// `transaction.run` will do all the same checks, so this step is not
+// necessary, but it may be useful in some workflows.
+std::cout << std::endl << "Verifying the transaction" << std::endl;
+libdnf::base::Transaction::TransactionRunResult res = transaction.verify();
+if (res != libdnf::base::Transaction::TransactionRunResult::SUCCESS) {
+    std::cout << std::endl << "Verifying transaction failed!" << std::endl;
+    throw;
+}
+
```

Comment:
I'm not sure, whether including the `transaction.verify()` call to the tutorial is a good choice.
The use case for the dry-run is rather marginal and once it gets into the tutorial, I'm afraid that people will start copying it to their projects which is unnecessary in most cases (or even harmful in terms of performance).

Reply:
Totally, I was not planning to keep it there. Looks like `dnf-behave-tests/dnf/steps/cmd.py` is the place as Eva said.